snyk-monitor/README.md

```diff
@@ -128,6 +128,31 @@ If '--reset-values' is specified, this is ignored.
 
 If running with Operator Lifecycle Manager (OLM) then OLM will handle upgrades for you when you request to install the latest version. This applies to OpenShift (OCP) and regular installations of OLM.
 
+## Sysdig Integration ##
+
+We have partnered with Sysdig to enrich the issues detected by Snyk for workloads with runtime data provided by Sysdig.
+
+In order for the integration with Sysdig to work, the Snyk monitor requires an extra Secret in the `snyk-monitor` namespace. The Secret name is `sysdig-eve-secret`.
+
+Please refer to the [Sysdig Secret installation guide](<here>) to install the Secret. Once the Sysdig Secret is installed, you need to copy it over to the snyk-monitor namespace:
+
+```bash
+kubectl get secret sysdig-eve-secret -n sysdig-agent -o yaml | grep -v '^\s*namespace:\s' | kubectl apply -n snyk-monitor  -f -
+```
+
+To enable Snyk to integrate with Sysdig and collect information about packages executed at runtime, use `--set sysdig.enabled=true` when installing the snyk-monitor:
+
+```bash
+helm upgrade --install snyk-monitor snyk-charts/snyk-monitor \
+  --namespace snyk-monitor \
+  --set clusterName="Production cluster" \
+  --set sysdig.enabled=true
+```
+
+> NOTE: The above command should be executed right after installing Sysdig to allow the snyk-monitor to detect Sysdig in the cluster.
```

Comment:
`> NOTE: The above command should be executed right after installing Sysdig. This will upgrade or install the snyk monitor, to allow the detection of Sysdig in the cluster.`